src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -38,19 +56,20 @@ export default function SearchResult({ form }) {
       </dd>
 
       <dd className="vads-u-padding-bottom--3">
-        <a
-          href={form.attributes.url}
-          rel="noopener noreferrer"
-          target="_blank"
-          download
+        <button
+          className="va-button-link"
+          onClick={onDownloadClick(form.attributes.url)}
```

Comment:
I don't think _any_ browsers support the `download` attribute how we are expecting it to behave (where it just downloads to the tray instead of redirecting or opening a new tab)

Reply:
Ah I know what it was now! When the `download` link is cross-origin, Chrome won't download it but instead proceed with the redirect, so locally it doesn't download to the tray.